index.bs

```diff
@@ -119,6 +119,19 @@ To <dfn>Construct an AmbientLightSensor Object</dfn> the user agent must invoke
 The <a attribute for="AmbientLightSensor">illuminance</a> attribute of the {{AmbientLightSensor}}
 interface represents the <a>current light level</a>.
 
+Use Cases and Requirements {#usecases-requirements}
+=========
+
+1. A Web application gradually updates document style based on light level changes.
+1. A Web application provides input for a smart home system to control lighting.
+1. A Web aplication checks whether light level at work space is sufficient, based on occupational
```

Comment:
There is only one constraint that comes to my mind. Usually, fov for the mobile sensors is limited, that's why, e.g., flash / light meters (that I use, quite often), have omni-directional meter in addition to spot meter. App can fix this limitation, e.g., use sensors + als and ask user to tilt device slightly, to capture als readings for larger fov area.

10 Lux resolution that is guaranteed by sensor manufacturers, should be enough to measure illuminance for workplace.